src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+export const CATEGORY_ACTION = {
+  STATUS: 'category_status',
+};
+
```

Comment:
- [ ] Same here, As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Define%20an%20action%20type%20for%20checking%20the%20status.), Please define an action type for checking the status using [ducks pattern](https://github.com/erikras/ducks-modular-redux).